airflow/www/security_manager.py

```diff
@@ -355,3 +347,20 @@ def _is_authorized_category_menu(self, category: str) -> Callable:
         return lambda action, resource_pk, user: any(
             self._get_auth_manager_is_authorized_method(fab_resource_name=item) for item in items
         )
+
+    """
+    The following methods are specific to FAB auth manager. They still need to be "present" in the main
+    security manager class, but they do nothing.
+    """
+
+    def get_action(self, name: str) -> Action:
+        raise NotImplementedError()
```

Comment:
Should there be a message with some context here?  The code that was moved into here used to have one.

Reply:
I can add one. This is something that should never happen though, that's why I did not specify any message but it makes no harm adding one